### DISCUSSION TITLE
Image Size

### DISCUSSION THREAD

**Discussion by kookma** [kookma/TW-Tamasha#15] [Q&A]
The current width of slide in full screen is limited to 900px which is good for most real cases specially those projected on wall.
By the way there is overflow in some cases reported in forum. One solution is to have a max-width for images as below

```
.tamasha-slide-content img{
    max-width:100%;
}
```

Automatic scaling may affect aspect ratio or enlarge images to sizes larger than the original image size and cause quality loss.

@NicolasPetton - what do you recommend here? 

I testes image macros using image macro in Shiraz and they work fine. The default value for an inserted image is `width:30%`

**Comment by NicolasPetton** [comment]
> Automatic scaling may affect aspect ratio or enlarge images to sizes larger than the original image size and cause quality loss.

Indeed, aspect ratio must be maintained.  Using `max-width` seems appropriate, unless specific problems with image sizes arise, so I'd go with that as a first approach as well :+1:
